helm/csi-powermax/values.yaml

```diff
@@ -164,19 +164,32 @@ controller:
   # Allowed values: map of key-value pairs
   # Default value: None
   # Examples:
-  #   node-role.kubernetes.io/control-plane: ""
+  # Uncomment if nodes you wish to use have the node-role.kubernetes.io/master taint
```

Comment:
do we want to specify upto which K8s version this true?

Reply:
hi @nidtara  , this is irrespective of kubernetes version . user can whichever role is available .